i_have_no_vinaigrette.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+import random
+
+
+def get_random_date():
+    """
```

Comment:
Nice documentation! good job! 
read the relevant parts of this document - add type hints and return types

Reply:
![Capture](https://user-images.githubusercontent.com/13467749/159839801-546a5447-18fb-4142-aafd-4c18b5309b8c.JPG)
should look like this